### ISSUE TITLE
Audio devices in System Preferences are in Russian

### ISSUE THREAD

**Issue by mihir3535** [toleda/audio_CloverALC#17]
Using the 1.0.4 version of the script with Clover v3128 set up for UEFI installation, all my audio is working as expected, but the devices in System Preferences and System Profiler are in Russian. I've tried to undo everything and re-install the CloverALC patches via the script, but I still get the same result. Previously I had the Multibeast AppleHDA.kext installed which showed English names.

![screen shot 2015-01-15 at 12 11 07 am](https://cloud.githubusercontent.com/assets/5190697/5752785/046f75bc-9c4b-11e4-99b7-4fcbfd9f3171.png)

**Comment by toleda** [comment]
To be clear, the script does not touch localization. Likely, installed AppleHDA.kext was not native as required. See README/A. Requirements/2. Native AppleHDA.kext, click: Need native. Once installed, run script again.

**Comment by mihir3535** [comment]
Yeah, it's strange because to prepare my machine for this script, I did

     sudo rm -rf /System/Library/Extensions/AppleHDA.kext/

then I installed the OS X 10.10.1 update again from [support.apple.com](http://support.apple.com/kb/DL1779) (making sure that the language at the top was English) and once I rebooted, ran the script, and rebooted once more, it was in Russian. Perhaps Apple accidentally provided the wrong localization of AppleHDA.kext in the English DMG, or there's something else I forgot to remove.

In any case, I ended up copying the stock AppleHDA.kext from my MacBook Air, re-running the script, and now everything is working on my hackintosh with English. Thank you for all your help and hard work Toleda!
